lib/statsd_middleware.rb

```diff
@@ -3,6 +3,52 @@
 class StatsdMiddleware
   STATUS_KEY   = 'api.rack.request'
   DURATION_KEY = 'api.rack.request.duration'
+  SOURCE_APP_NAMES = %w[
+    0993-edu-benefits
+    0994-edu-benefits
+    1990-edu-benefits
+    1990e-edu-benefits
+    1990n-edu-benefits
+    1995-STEM-edu-benefits
+    1995-edu-benefits
+    526EZ-all-claims
+    526EZ-claims-increase
+    5490-edu-benefits
+    5495-edu-benefits
+    686-dependent-status
+    account
+    auth
+    beta-enrollment
+    burials
+    chapter31-vre
+    chapter36-vre
+    claims-status
+    connected-accounts
+    dashboard
+    discharge-upgrade-instructions
+    facilities
+    feedback-tool
+    gi
+    hca
+    id-card-beta
+    letters
+    my-health-account-validation
+    pensions
+    post-911-gib-status
+    pre-need
+    profile-360
+    profile-360-beta
+    proxy-rewrite
+    public-outreach-materials
+    search
+    static-pages
+    terms-and-conditions
+    vaos
+    verify
+    veteran-id-card
+    veteran-representative
+    vic-v2
```

Comment:
@cvalarida I got this list by looking in vets-website for all manifest.json files and finding the line:
```
"entryName": "foo"
```
Do you know if this list is complete?

Reply:
That's as complete as I think we can get right now. You can verify this by running `yarn apps` or (`npm run apps`). That'll do the same thing.

Reply:
Created a ticket for this: https://github.com/department-of-veterans-affairs/va.gov-team/issues/4421